### ISSUE TITLE
some questions about codes and network

### ISSUE THREAD

**Issue by Z-Lee-corder** [xmuqimingxia/3D-HANet#5]
Thank you very much for sharing the code，However, I have two questions:
1. This paper adopts an attention adaptive module to adjust the weighting parameters of different loss. But I didn't find this part in the code, and the weights of each loss seem to be predefined. For the heat map loss, when calculating this part in the code, multiply it by a weight of "5", which is  “hm_loss = 5 * (hm_loss_1_N + hm_loss_2_N)/2” in“anchor_head_template.py”. Other losses also use pre-defined weights.
2. The whole process from the heatmap prediction to the calculation of heatmap Loss function does not seem to be interacted with backbone network. That is to say, the backbone network has not been enhanced by the heat map prediction network in the whole process of feature extraction and final box prediction. Therefore, I do not understand where the performance improvement comes from.  In my opinion, at least one of the three parts "backbone network, neck, and head" should be helped by  heatmap prediction network (such as features enhancement). Because the heat map loss seems to only help the network predict the real heatmap better, it does not seem to have gain on the box prediction branch.

**Comment by xmuqimingxia** [comment]
Thanks for your questions.
1，From the ablation experiments of the paper, the role of AAW （attention adaptive module）diminishes as the network becomes more complex. In order to expedite the convergence of the network, we directly defined weights for the multi-stage CasA detector.

2，Our 3D heatmap auxiliary network is involved in the entire detection network training process. During the process of gradient backpropagation, the parameters of the backbone network are influenced by the auxiliary network's task. Specifically, the auxiliary network utilizes features outputted by the backbone network, and it can only converge effectively when the features from the backbone network have high discriminative power.

**Comment by Z-Lee-corder** [comment]
Thanks for your reply sincerely！
